trunner/target/armv7m4.py

```diff
@@ -93,7 +86,7 @@ def __call__(self):
         offset = self.load_offset
         for app in self.apps:
             path = self.gdb.cwd / Path(app.file)
-            sz = self._app_size(path)
+            sz = path.stat().st_size
```

Comment:
in line 91  it;s good, but in line 94 you need to align to page

Reply:
Oh right, sorry for that, it's now fixed.

Reply:
I can't add comment in place, because is too far in context, but still issue, now in line 101.

Offset needs to be page aligned. Only size in alias command shouldn't be aligned to provide valid binary size to kernel.

Reply:
Yeah, now I see, thanks for being watchful. I hope it won't be a big issue to get file size two times (lines 96 and 101)

Reply:
It's good enough